accounts/serializers.py

```diff
@@ -50,6 +50,8 @@ class Meta:
             'city',
             'state',
             'zip_code',
+            # 'latitude',
+            # 'longitude',
```

Comment:
please consider uncommenting lat & lon - when I uncommented them, it worked as expected in Postman
```suggestion
            'latitude',
            'longitude',
```